pkg/app/piped/platformprovider/ecs/ecs.go

```diff
@@ -19,13 +19,18 @@ import (
 	"path/filepath"
 	"sync"
 
+	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/ecs/types"
 	"go.uber.org/zap"
 	"golang.org/x/sync/singleflight"
 
 	"github.com/pipe-cd/pipecd/pkg/config"
 )
 
+const (
+	LabelApplication string = "pipecd-dev-application" // The application this resource belongs to.
```

Comment:
Could you add more builtin tags for ECS resources. For instant, in case of CloudRun, we have as below
https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/platformprovider/cloudrun/cloudrun.go#L86-L93

I think we should also have: TagManagedBy (value will be piped in all cases), LabelPiped, LabelCommitHash as the PipeCD builtin tags for ECS resources.